NEWS.md

```diff
@@ -71,6 +71,22 @@
   indefinitely; `scoped_options()` and `with_options()` change options
   temporarily.
 
+* New function `meow()`. It is equivalent to `cat()` but interleaves
```

Comment:
I have unexported these functions. Should the cli functions be plural, e.g. `cat_lines()`, `cat_bullets()`? Also should their arguments be prefixed with dots (just for consistency since they don't take named arguments through `...`)?

Reply:
I think we usually go with singular names for vectorised functions?

Reply:
`cat_line()` is not really vectorised, it accepts a variable number of arguments. In rlang we have used the plural in these cases, e.g. `with_handlers()` or `env_bind_exprs()`.

Reply:
And `...` is just temporary until the API stabiliises

Reply:
Oh `cat_line()` different - it both uses `...` and is vectorised